lib/commands/wrappers/schedule.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Commands
+  module Wrappers
+    class Schedule < Base
```

Comment:
is this a Commands::Base? i guess wrappers should not be inherited from Command.

Reply:
Why do you use for this class namespace `Wrappers`? I guess it's should be named as `Collections`...